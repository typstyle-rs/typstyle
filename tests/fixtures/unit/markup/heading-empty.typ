=     
==   //   
===    /*  */   
= <label>
  =  <label>
  = /*
 */<label>
= /*  */ <label>
  ==/*  */<label>
   ==  /*
 */  <label>
==  a  <label>
   ===  b//
== c  //
  = d /*  */  <label>
==  /*  */ e<label>
= f /*
 */  <label>
  ==  /*
 */ g <label>
