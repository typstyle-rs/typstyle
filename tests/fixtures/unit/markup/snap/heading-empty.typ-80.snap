---
source: tests/src/unit.rs
input_file: tests/fixtures/unit/markup/heading-empty.typ
---
=
==   //
===    /*  */
= <label>
=  <label>
= /*
   */<label>
= /*  */ <label>
==/*  */<label>
==  /*
     */  <label>
== a  <label>
=== b//
== c  //
= d /*  */  <label>
== /*  */ e<label>
= f /*
     */  <label>
== /*
    */ g <label>
